admin_guide/manage_authorization_policy.adoc

```diff
@@ -0,0 +1,357 @@
+= Managing Authorization Policy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use link:../cli_reference/overview.html[the CLI] to view
+link:../architecture/additional_concepts/authorization.html[authorization
+policy] and the administrator CLI to manage the
```

Comment:
@adellape Should this be "to view _the_ authorization policy" or "to view authorizations _policies_"?